src/main/java/dao/FruitDao.java

```diff
@@ -0,0 +1,12 @@
+package dao;
+
+import java.util.Map;
+import model.Fruit;
+
+public interface FruitDao {
+    void add(Fruit fruit, Integer amount);
+
+    Integer get(Fruit fruit);
+
+    Map<Fruit, Integer> getDB();
```

Comment:
this method looks strange in DAO layer, why do we need it if just can getDB and you put and get directly on Map. I would replace it with getAll() method to return all data if we need smt like this in code